README.md

```diff
@@ -1,30 +1,163 @@
-# Edu Client
+<h1 align="center">
+    ✨ Edu Client ✨
+</h1>
 
-Product focuses on 100% education as well as upskilling developing countries and rural areas. 
+<blockquote align="center"> <b><i>
+    Product focuses on 100% education as well as upskilling developing countries and rural areas.</i></b>
+</blockquote>
 
-## Available Scripts
+## ❤️ Note to new contributors 
 
-In the project directory, you can run:
+Thank you for considering and taking the time to contribute! Before contributing kindly read and follow [Code of conduct](CODE_OF_CONDUCT.md). To help new developers/contributors there are set of instructions added in Contributing.md. Go ahead and read it [here](CONTRIBUTING.md). Which describes the intial stages for working on this project. Also refer the [MIT License](LICENSE.txt).
```

Comment:
Change `LICENSE.txt` to `LICENSE`

Reply:
Resolved By @kunal-kushwaha in merged PR #49